docs/cloud-native-security/cloud-workload-protection.asciidoc

```diff
@@ -1,15 +1,15 @@
 [[cloud-workload-protection]]
-= Cloud workload protection
+= Cloud workload protection (VMs)
```

Comment:
This addition seemed a little unclear when you see it in the side TOC, out of context. Maybe:
```suggestion
= Cloud workload protection for VMs
```